### PR TITLE
Add context properties 

### PR DESCRIPTION
**Pull request by rapidoo**
In aim to facilite apache proxy on Azkaban, add a propertie web.context=/azkaban

### CODE REVIEW

File: azkaban-webserver/src/main/java/azkaban/webapp/AzkabanWebServer.java [hunk 1]
```diff
@@ -759,6 +759,10 @@ public static void main(String[] args) throws Exception {
     String staticDir =
         azkabanSettings.getString("web.resource.dir", DEFAULT_STATIC_DIR);
     logger.info("Setting up web resource dir " + staticDir);
+
+    String contextDir =
+        azkabanSettings.getString("web.context", "/");
```
Comment (davidzchen) [inline azkaban-webserver/src/main/java/azkaban/webapp/AzkabanWebServer.java]: How about `web.site_root` instead? That might be more descriptive.

Comment (rapidoo) [inline azkaban-webserver/src/main/java/azkaban/webapp/AzkabanWebServer.java]: Sure, please replace the var and the properties 

Fred

**Comment by rapidoo** [comment]
Sorry I miss to use the var in Context ...

**Comment by rapidoo** [comment]
replace by https://github.com/azkaban/azkaban/pull/579

**Pull request closed without merging**